src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -438,8 +447,17 @@ describe("Routes/ConfirmBid", () => {
       await page.agreeToTerms()
       await page.submitForm()
 
-      expect(mockPostEvent).toHaveBeenCalledTimes(1)
-      expect(mockPostEvent).toBeCalledWith({
+      expect(mockPostEvent).toHaveBeenCalledTimes(2)
+      expect(mockPostEvent.mock.calls[0][0]).toEqual({
```

Comment:
oh this is interesting, so we're simulating the case where registration succeeds, but bidding fails. I'd recommend updating the docstring here to make this more clear.

Reply:
Also, are we testing the case when registration fails when attempting to bid+register?

Reply:
We can't track registration failed because it's all done [by Gravity](https://github.com/artsy/gravity/blob/5985f0f43078699d7ff81d963e08286f6d3db32f/app/api/v1/bidder_positions_endpoint.rb#L65-L72) in the single bidder position endpoint and errors from Gravity will be translated to a GraphQL error in Metaphysics.